meteor/client/styles/shelf/dashboard.scss

```diff
@@ -616,4 +616,95 @@ $dashboard-button-height: 5.625em;
 			}
 		}
 	}
-}
+	
+	&.dashboard-panel__panel--bucket {
+		$dashboard-button-width: 10.625em;
+		$dashboard-button-height: 8.90625em;
+		$dashboard-button-label-height: 2.7em;
+		$dashboard-button-thumbnail-height: $dashboard-button-height - $dashboard-button-label-height;
+
+		.dashboard-panel__panel__group {
+	
+			.dashboard-panel__panel__group__liveline {
+				transform: translate(calc(-1 * #{$dashboard-button-width}), 0);
+				&::after {
+					transform: translate(calc(#{$dashboard-button-width} + -1px), 0);
+				}
+	
+				&::before {
+					transform: translate(calc(#{$dashboard-button-width} + -1px), 0);
+				}
+			}
+		}
+
+		.dashboard-panel__panel__button {
+			border-radius: 0px;
+			width: $dashboard-button-width;
+			height: $dashboard-button-height;
+
+			&.source-missing, &.source-missing, &.unknown-state {
+				&::before {
+					z-index: 2;
+				}
+			}
+
+			> .dashboard-panel__panel__button__content {
+				background: linear-gradient(90deg, rgb(84,84,84) 0%, rgb(61,61,61) 100%);
+			
+
+				> .dashboard-panel__panel__button__label-container {
+					background-color: #000;
+					height: $dashboard-button-label-height;
+					display: flex;
+					position: absolute;
+					top: $dashboard-button-thumbnail-height;
+					right: unset;
+					bottom: unset;
+					left: unset;
+					padding: unset;
```

Comment:
This seems weird? If the label is supposed to be attached to the bottom of the button, why position it from the top? Wouldn't it make more sense to just set `top: auto; bottom: 0`? Also, I think the label was supposed to span the entire width of the button, so I think this should be `left: 0; right: 0` as well, so that it looks like this?

![obraz](https://user-images.githubusercontent.com/3635991/218788784-c67ad7bf-26ab-49d2-b024-b3d8a7424be8.png)

If not, and the label is supposed to be a left-aligned flag, the `min-width` on `.dashboard-panel__panel__button__label` should be set to `fit-content` so that it doesn't look like this:

![obraz](https://user-images.githubusercontent.com/3635991/218789074-9168b512-9508-49da-b395-20d29193d359.png)

but instead looks like this:

![obraz](https://user-images.githubusercontent.com/3635991/218789303-2d70b2bb-1f28-4037-bd43-84773038be22.png)

In which case we would need a right padding value, I think.